xrviz/dashboard.py

```diff
@@ -339,14 +340,13 @@ def create_indexed_graph(self, *args):
         # 0,1(min, max) to get a color balance graph
         c_lim_lower, c_lim_upper = (
             (float(cmin), float(cmax)) if cmin and cmax
-            else ([q for q in sel_data.quantile([0.1, 0.9])])
-        )
+            else find_cmap_limits(sel_data))
 
         color_range = {sel_data.name: (c_lim_lower, c_lim_upper)}
 
         if not cmin:  # if user left blank or initial values are empty
-            self.control.style.lower_limit.value = str(c_lim_lower.values.round(5))
-            self.control.style.upper_limit.value = str(c_lim_upper.values.round(5))
+            self.control.style.lower_limit.value = str(round(c_lim_lower, 5))
```

Comment:
Without rounding, the float values can be upto 16 decimal places (`5.454999923706055`) which might not look good, so just for presentation purposes I am rounding off to 5.
This is not necessary and can be removed.

Reply:
It may be right, but we should own that it is an arbitrary decision.

Reply:
I will mention about this in docs.